src/packet/secret_key.js

```diff
@@ -493,6 +506,10 @@ class SecretKeyPacket extends PublicKeyPacket {
       throw new Error('Key is not decrypted');
     }
 
+    if (this.skipValidation) {
+      return true;
```

Comment:
The function normally doesn't return a bool so we shouldn't here either.
```suggestion
      return;
```